packages/web5/test/dids/did_web_test.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:test/test.dart';
+import 'package:web5/src/dids/did_web/did_web.dart';
+
+void main() {
+  group('DidWeb', () {
+    test('should work', () async {
```

Comment:
`should resolve`? Or maybe `should resolve successfully`?

Reply:
Haha yeah good point. Was so tired last night. Will change to should resolve successfully